ipfs-api/src/client.rs

```diff
@@ -821,6 +821,19 @@ impl IpfsClient {
         self.request(&request::ObjectStat { key }, None)
     }
 
+    /// Pins a new object.
+    ///
+    /// Does not yet implement "progress" because reading it is kinda squirrelly.
+    ///
+    #[inline]
+    pub fn pin_add(
+        &self,
+        key: &str,
+        recursive: Option<bool>
```

Comment:
Can you change this from `Option<bool>` type to just be a `bool` instead. I just made the change with some other calls!